app/models/batch.rb

```diff
@@ -567,10 +568,6 @@ def request_count
     requests.count
   end
 
-  def show_actions?
-    released? == false or pipeline.class.const_get(:ALWAYS_SHOW_RELEASE_ACTIONS)
-  end
-
```

Comment:
We now always show the actions, but disable them on a case by case basis. (With a nice little message explaining why they are disabled)